<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox</class>
 <widget class="QWidget" name="MainBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QSpinBox" name="sb_max_wait"/>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="sb_src_b"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>End</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="sb_dst_a"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Max wait, ms</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Begin</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="sb_src_c"/>
     </item>
     <item row="0" column="4">
      <widget class="QSpinBox" name="sb_src_d"/>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="sb_dst_b"/>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="sb_src_a"/>
     </item>
     <item row="1" column="4">
      <widget class="QSpinBox" name="sb_dst_d"/>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="sb_dst_c"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="btn_connect">
       <property name="text">
        <string>connect</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="sb_port"/>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="btn_disconnect">
       <property name="text">
        <string>disconnect</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QPushButton" name="btn_test">
       <property name="text">
        <string>test</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
